resdk/scripts.py

```diff
@@ -481,3 +484,74 @@ def update_knowledge_base():
         logger.warning("Encountered {} errors during import.".format(errors))
 
     logger.info("Updated {} features.".format(updated))
+
+
+def upload_vaccinesurvey_data():
+    """
+    Upload vaccinesurvey samples from a .tsv file.
+
+    Usage::
+
+        resolwe-upload-vcsrvy -a www.genialis.com -e admin -p admin -f data.csv
+
+    """
+    parser = argparse.ArgumentParser(description="Upload vaccinesurvey samples")
+    parser.add_argument('-f', '--file', required=True, help='File with data')
+    parser.add_argument('-a', '--address', help='Resolwe server address')
+    parser.add_argument('-e', '--email', default='admin', help='User name')
+    parser.add_argument('-p', '--password', default='admin', help='User password')
+    parser.add_argument('-v', '--verbose', action='store_true', help='Enable verbose reporting')
+
+    args = parser.parse_args()
+
+    if args.verbose:
+        resdk_logger.start_logging()
+
+    res = Resolwe(args.email, args.password, args.address)
+
+    strings = ['study_code', 'sex', 'ethnicity']
+    ints = ['village_code']
+    floats = ['body_temp']
+    dates = ['birth_date', 'entry_date']
+    booleans = ['fever', 'antimalaria_treatment', 'hospital_visit', 'vomit',
+                'cough', 'diarrhoea', 'bednet']
+    immunological = ['ama1', 'msp1', 'msp2', 'nanp', 'total_ige', 'haemoglobin']
+
+    with open(args.file, 'rt') as ofile:
```

Comment:
I suggest reading tab separated files with a [DictReader](https://docs.python.org/2/library/csv.html#csv.DictReader). It's more robust.